Update details blog

theshoplondon opened the pull request.

theshoplondon committed:
Create updating-member-details

theshoplondon committed:
Rename updating-member-details to 2024-08-27-updating-member-details

theshoplondon committed:
Update 2024-08-27-updating-member-details

rgarner committed:
Add .markdown ext to update details post

rgarner commented on _posts/2024-08-27-updating-member-details.markdown:
```diff
@@ -0,0 +1,50 @@
+
+### Try it for yourself
+
+We’ve put together a [demo](https://https://jtreview-pr-1167.herokuapp.com) so you can see how it works. Just add your email (your details won’t be stored) to get started, then try the update process for yourself.
```
We can't currently say

> (your details won’t be stored)

As, y'know, they have to be, sort of.

rgarner commented on _posts/2024-08-27-updating-member-details.markdown:
```diff
@@ -0,0 +1,50 @@
+
+The Join Together approach to improving your ‘update your details’ process will lead to:
+
+- Better, more frequent contact and engagement with members
```
I'd drop this a couple of places. Revenue and retention are our top two.

rgarner commented on _posts/2024-08-27-updating-member-details.markdown:
```diff
@@ -0,0 +1,50 @@
+
+Each of these has different challenges and costs. Calls and letters are expensive to make or send, get low response rates and often interrupt what members are doing at a particular moment (”I’ll do that later” and “who’s that number calling me?”). 
+
+Collecting information via spreadsheets leads to enormous data protection risks, as well as problems with data entry, as files are passed from person to person, computer to computer, usually ending with a messy import or copy/paste/fix the mistakes process, slowing the whole thing down.
```
Is this genuinely still a thing?

rgarner commented on _posts/2024-08-27-updating-member-details.markdown:
```diff
@@ -0,0 +1,50 @@
+
+Collecting information via spreadsheets leads to enormous data protection risks, as well as problems with data entry, as files are passed from person to person, computer to computer, usually ending with a messy import or copy/paste/fix the mistakes process, slowing the whole thing down.
+
+Finally, union membership portals are challenging for their users. The typical experience involves using data members probably don’t know (their membership number or a password they never use, leading to a password reset or an email to the membership team) before logging into a piece of software they’re unfamiliar with to perform a task they can’t be sure they’ll be able to complete.
```
This is sweet. They read well in this order (calls/letters->spreadsheets->portals). Is this the order that's most common -> least common?

rgarner reviewed (commented):
Looks good, I think. It looks like it speaks directly to some problems that I can imagine existing, but I don't know if they do in 2024 so I'm taking your word for that

tekin commented on _posts/2024-08-27-updating-member-details.markdown:
```diff
@@ -0,0 +1,50 @@
+date:   2024-08-27 11:19:13 +0000
+---
+
+We’ve worked with unions for years, and heard all sorts of horror stories about the difficulties of staying in touch with members. To try and fix this, we’ve seen unions taking a variety of approaches. These include:
```
Are there really "Horror stories" about "...staying in touch with members"? If there are, should we say what they are? Or are the horror stories actually about trying to keep member data up to date?

tekin committed:
Suggested adjustments/edits to update-your-deets

rgarner committed:
Content pass to updating-member-details

- fix some hyphens
- fix run-on sentences tense and capitalisation in bullets
- shorten long sentences with a lot of commas
- reduce a small amount of repetition around "pain" and "surprise"
